chart-tests/application/ci/test-image-lifecycle-values.yaml

```
@@ -9,6 +9,7 @@ image:
   repository: busybox
   tag: stable
 container:
+  command: ["sh", "-c"]
   args: ["sleep", "2000"]
```

```suggestion
  args: ["sleep", "2"]
```

sleep takes seconds :P